src/bp/ui-studio/src/web/views/FlowBuilder/diagram/index.tsx

```diff
@@ -315,6 +315,8 @@ class Diagram extends Component<Props> {
 
   add = {
     flowNode: (point: Point) => this.props.createFlowNode({ ...point, type: 'standard', next: [defaultTransition] }),
+    commentNode: (point: Point) =>
+      this.props.createFlowNode({ ...point, type: 'comment', next: [], isResizable: true, width: 200, height: 40 }),
```

Comment:
If you only make comment nodes resizable, why add a property on the node itself? I suggest just having `const isResizable = node.type === 'comment'` or even better, make a const somewhere `const resizableNodeTypes = ['comment']`